src/Components/Modal/Content/ExitMedicalCase.js

```diff
@@ -1,10 +1,11 @@
 /**
  * The external imports
  */
-import React from 'react'
-import { View, Text } from 'react-native'
+import React, { useEffect } from 'react'
+import { View, Text, BackHandler } from 'react-native'
```

Comment:
Do you even use useEffect and BackHandler or are you a dumdum and you forgot to remove them ?